_scripts/instruction-widget/templates/install/pip.html

```diff
@@ -0,0 +1,81 @@
+<aside class="note">
+  <div class="note-header">
+    <h3>Partial support</h3>
+  </div>
+  <p>
+  The Certbot team supports this installation method on a best effort basis. If you are on
+  a more obscure or heavily customized system, these instructions may not work and the
+  Certbot team may be unable to help you resolve the problem.
+  </p>
+</aside>
+
+{{> header}}
+<li>
+    Install system dependencies
+
+    <p>System dependencies may include Python 3.6+, including venv (this might be called <code>python3-dev</code>,
+      <code>python3-devel</code>, or <code>python3-venv</code>) and Augeas for the Apache
+      plugin (<code>libaugeas0</code> or <code>augeaslibs</code>).</p>
+
+    <p>If you're having trouble installing cryptography, you may need to install additional
+      dependencies. See
+      <a href="https://cryptography.io/en/latest/installation.html#building-cryptography-on-linux">
+      the cryptography project's site</a> for more infoformation.</p>
+
+    <p>Commands to install system dependencies may look like the following, run on the command
+    line on the machine:
+      <pre class="no-before"><ol><li># For APT-based distributions (e.g. Debian, Ubuntu ...)</li>
+        <li>sudo apt update</li>
+        <li>sudo apt install python3-dev python3-venv libaugeas0</li>
+        <li># For RPM-based distributions (e.g. Fedora, CentOS ...)</li>
+        <li># NB1: old distributions will use yum instead of dnf</li>
+        <li># NB2: RHEL-based distributions use python3X-devel instead of python3-devel (e.g. python36-devel)</li>
+        <li>sudo dnf install python3-devel augeas-libs redhat-rpm-config</li></ol></pre>
+    </p>
+
+    <p>These packages may have slightly different names on other distributions; a search such as
+    "augeas on &lt;your_system_name&gt;" will probably yield helpful results.</p>
+</li>
+<li>
+  Set up a venv
+  <p>
+  Execute the following instructions on the command line on the machine to set up a virtual
+  environment.
+  <pre class="no-before"><ol>
+        <li>sudo python3 -m venv /opt/certbot/</li>
+        <li>{{install_command}} --upgrade pip</li>
+  </ol></pre>
+  </p>
+</li>
+
+{{> installcertbot}}
```

Comment:
Like we do in our snap instructions, somewhere above this point I think we may want to encourage people to uninstall any Certbot OS packages. What do you think?

Reply:
great call!